### ISSUE TITLE
hashicorp/google 4.0.0 broke the google_compute_region_instance_group_manager resource

### ISSUE THREAD

**Issue by lonnblad** [hashicorp/terraform-google-consul#66]
In the 4.0.0 release of hashicorp/google, the beta property `min_ready_sec` was removed.
https://github.com/hashicorp/terraform-provider-google/pull/10410

This broke the execution of this repository.

**Comment by TonyRoussel** [comment]
FYI, this field is still mentionned in [official documentation](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance_group_manager)

**Comment by lonnblad** [comment]
> FYI, this field is still mentionned in [official documentation](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance_group_manager)

Yep, I know, the doc is not aligned with what the code actually supports.
